.drone.yml

```diff
@@ -274,11 +274,124 @@ trigger:
       - promote
 ---
 kind: pipeline
+name: linux-amd64
+platform:
+  os: linux
+  arch: amd64
+steps:
+  - name: build
+    image: rancher/dapper:v0.5.6
+    commands:
+      - "docker pull fluent/fluent-bit:x86_64-${DRONE_TAG}"
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+    when:
+      event:
+        - tag
+  - name: docker-publish
+    image: plugins/docker
+    settings:
+      custom_dns: 1.1.1.1
+      username:
+        from_secret: docker_username
+      password:
+        from_secret: docker_password
+      repo: "rancher/fluent-bit"
+      tag: "${DRONE_TAG}-linux-amd64"
+    when:
+      event:
+        - tag
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+trigger:
+  event:
+    exclude:
+      - promote
+---
+kind: pipeline
+name: linux-arm64
+platform:
+  os: linux
+  arch: arm64
+steps:
+  - name: build
+    image: rancher/dapper:v0.5.6
+    commands:
+      - "docker pull fluent/fluent-bit:arm64v8-${DRONE_TAG}"
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+    when:
+      event:
+        - tag
+  - name: docker-publish
+    image: plugins/docker
+    settings:
+      custom_dns: 1.1.1.1
+      username:
+        from_secret: docker_username
+      password:
+        from_secret: docker_password
+      repo: "rancher/fluent-bit"
+      tag: "${DRONE_TAG}-linux-arm64"
+    when:
+      event:
+        - tag
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+trigger:
+  event:
+    exclude:
+      - promote
+---
+kind: pipeline
+name: linux-arm
+platform:
+  os: linux
+  arch: arm
+steps:
+  - name: build
+    image: rancher/dapper:v0.5.6
+    commands:
+      - "docker pull fluent/fluent-bit:arm32v7-${DRONE_TAG}"
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+    when:
+      event:
+        - tag
+  - name: docker-publish
+    image: plugins/docker
+    settings:
+      custom_dns: 1.1.1.1
+      username:
+        from_secret: docker_username
+      password:
+        from_secret: docker_password
+      repo: "rancher/fluent-bit"
+      tag: "${DRONE_TAG}-linux-arm"
+    when:
+      event:
+        - tag
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+trigger:
+  event:
+    exclude:
+      - promote
+---
+kind: pipeline
 name: manifest
 platform:
-  os: windows
+  os: linux
```

Comment:
Note: there's no need to use Windows for the manifest, so we default to linux-amd64.